### ISSUE TITLE
Use SQS batch requests where possible

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#1545]
Let's use [batch requests](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_Message.html) such as `AmazonSQS.deleteMessageBatch()`, `AmazonSQS.sendMessageBatch()` to reduce SQS cost. When using `sendMessageBatch` we should ensure that size of **all** claims in a batch is less then `256KB`. I think it can be implemented as `class BatchClaims implements Claims, Closeable` which will accept all claims and keep them in memory, but send them on `close()`:
```java
try (final Claims claims = new BatchClaims()) {
  new ClaimOut().type("...").postTo(claims); // keep first claim
  new ClaimOut().type("...").postTo(claims); // keep second claim
} // send batch here (on close)
```
especially it can be helpfull for sending ping-claims in `Ping`

**Comment by 0crat** [comment]
Job [#1545](https://github.com/zerocracy/farm/issues/1545) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
The job [#1545](https://github.com/zerocracy/farm/issues/1545) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2262013499); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
@g4s8 2 puzzles [#1621](https://github.com/zerocracy/farm/issues/1621), [#1622](https://github.com/zerocracy/farm/issues/1622) are still not solved.

**Comment by paulodamaso** [comment]
@g4s8 merged, please close

**Comment by g4s8** [comment]
@paulodamaso thanks

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1545](https://github.com/zerocracy/farm/issues/1545) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#1621](https://github.com/zerocracy/farm/issues/1621) is still not solved; solved: [#1622](https://github.com/zerocracy/farm/issues/1622).

**Comment by 0pdd** [comment]
@g4s8 the puzzle [#unknown]() is still not solved.
